src/main/java/com/klarna/hiverunner/StandaloneHiveRunner.java

```diff
@@ -173,6 +173,7 @@ protected final void runTestMethod(FrameworkMethod method,
      */
     public HiveShellContainer evaluateStatement(List<? extends Script> scripts, Object target,
         Path temporaryFolder, Statement base) throws Throwable {
+        scripts = new ArrayList<>();
```

Comment:
This doesn't look right, you're nuking the value passed in as an argument. I'm surprised this doesn't cause a test to fail...